pkg/kymactl/cmd/version.go

```diff
@@ -25,8 +25,8 @@ func NewVersionOptions() *VersionOptions {
 func NewVersionCmd(o *VersionOptions) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "version",
-		Short: "Version of the kymactl and connected Kyma cluster",
-		Long: `Prints the version of kymactl itself and the version of the kyma cluster connected by current KUBECONFIG
+		Short: "Version of the kyma CLI and connected Kyma cluster",
```

Comment:
Maybe we should call it kyma client in the version context (versus kyma server)

Reply:
I'm ok with Kyma CLI (capitalisation btw), but I'm thinking about renaming whole project to `kyma-cli`. `kymactl` feels like binary name, but resulting binary is named differently.

Reply:
interesting point, let's talk about it after this PR. I think we should do it indeed.